dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEnrollmentAnalyticsTableManager.java

```diff
@@ -118,6 +94,7 @@ public JdbcEnrollmentAnalyticsTableManager(
         analyticsExportSettings,
         periodDataProvider,
         sqlBuilder);
+    FIXED_COLS = EnrollmentAnalyticsColumn.getColumns(sqlBuilder);
```

Comment:
Ok, the reason I didn't change it is that several tests are relying on this variable being static, but I agree that it's misleading since it's now populated in the constrructor.